backend/ingress/openapi.go

```diff
@@ -499,6 +499,12 @@ func schemaForType(t schema.Type) spec.Schema {
 				Type: []string{"object"},
 			},
 		}
+	case *schema.Any:
+		return spec.Schema{
+			SchemaProps: spec.SchemaProps{
+				Type: []string{"object"},
+			},
+		}
```

Comment:
@alecthomas I had to add this to make `just lint-backend` happy, but I'm not sure why I needed it in this PR 😬

Reply:
> "/Users/wesbillman/dev/ftl/backend/ingress/openapi.go:431:2: exhaustiveness check failed for sum type "Type" (from /Users/wesbillman/dev/ftl/common/schema/parser.go:107:6): missing cases for Any"